🐙 destination-pinecone: run up-to-date pipeline [2025-02-15]

airbyteio opened the pull request:
# Update destination-pinecone

This PR was autogenerated by running `airbyte-ci connectors --name=destination-pinecone up_to_date --pull`

We've set the `auto-merge` label on it, so it will be automatically merged if the CI pipelines pass.
If you don't want to merge it automatically, please remove the `auto-merge` label.
Please reach out to the Airbyte Connector Tooling team if you have any questions or concerns.


## Operations

- Upgrade the base image to the latest version in metadata.yaml: Successful

- Update versions of libraries in poetry.: Successful

- PATCH bump destination-pinecone version to 0.1.38: Successful

- Build destination-pinecone docker image for platform(s) linux/amd64, linux/arm64: Failed

- Create or update pull request on Airbyte repository: Successful

- Add changelog entry: Successful

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/airbytehq/airbyte?pullRequest=53879) <br/>All committers have signed the CLA.

airbyteio committed:
Upgrade the base image to the latest version in metadata.yaml
Update versions of libraries in poetry.
PATCH bump destination-pinecone version to 0.1.38
Create or update pull request on Airbyte repository
Add changelog entry
